src/savegame.cc

```diff
@@ -41,7 +41,7 @@
 #include <Knownfolders.h>
 #include <Shlobj.h>
 #include <direct.h>
-#elif defined(__APPLE__)
```

Comment:
Looks like this file have no any linkage to SDL.
Does it make sense to change anything here?